### PR TITLE
add german translations

### PR DESCRIPTION
**Pull request by derhedwig**

### CODE REVIEW

File: wagtailmedia/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+msgid "Add/edit media you own"
+msgstr "Hinzufügen/Editieren Ihrer Medien"
+
+#: forms.py:80 views/media.py:139 views/media.py:194
+msgid "Edit"
+msgstr "Bearbeiten"
+
+#: forms.py:80
+msgid "Edit any media"
+msgstr "Alle Medien bearbeiten"
```
Comment (Hopiu) [inline wagtailmedia/locale/de/LC_MESSAGES/django.po:36]: Without having a closer look at the context, wouldn't it be better translated with:
"Beliebige Medien bearbeiten"

File: wagtailmedia/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.4.2\n"
+
+#: forms.py:79
+msgid "Add"
+msgstr "Hinzufügen"
+
+#: forms.py:79
+msgid "Add/edit media you own"
+msgstr "Hinzufügen/Editieren Ihrer Medien"
```
Comment (Hopiu) [inline wagtailmedia/locale/de/LC_MESSAGES/django.po:28]: What about:
"Eigene Medien hinzufügen/bearbeiten"

**Review by Hopiu — COMMENTED** [review]
A few suggestions added

**Comment by zerolab** [comment]
Thank you @Hopiu !

File: wagtailmedia/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+
+#: views/media.py:136
+#, python-brace-format
+msgid "Media file '{0}' added."
+msgstr "Medien-Datei '{0}' hinzugefügt."
+
+#: views/media.py:146
+msgid "The media file could not be saved due to errors."
+msgstr ""
+"Die Medien-Datei konnte wegen Formular-Fehlern nicht gespeichert werden."
```
[the remark this answers is not in the document]
Comment (derhedwig) [reply wagtailmedia/locale/de/LC_MESSAGES/django.po:339]: Yeah, okay. Lets stick with the English message. I felt that "form error" was more helpful to the user, but if than it should be changed upstream...

**Comment by zerolab** [comment]
@Hopiu do let me know if you are happy with the changes and we can get this merged

**Review by Hopiu — APPROVED** [review]

**Review by zerolab — APPROVED** [review]

**Pull request merged**